src/Parsers/ASTColumnsTransformers.cpp

```diff
@@ -94,13 +95,26 @@ void ASTColumnsExceptTransformer::transform(ASTs & nodes) const
 {
     std::set<String> expected_columns;
     for (const auto & child : children)
-        expected_columns.insert(child->as<const ASTIdentifier &>().name());
+    {
```

Comment:
It is not obvious for me that this code is working correctly.
And there is no test for fully qualified identifier like `db.table.column`.
Maybe add more explanation in comments?

Reply:
At the beginning, I've thought to support database prefix. But I found there's no database info in the identifier of children parameter, So I don‘t know how to check weather it is a true database name, 
So, I just ignore database name.